contents/docs/data/data-warehouse.mdx

```diff
@@ -0,0 +1,118 @@
+
+6. Click "Next" until you end up on the "Review Policy" page
+7. Give your policy a name and click "Create policy"
+
+The final step is now to create a new user and give them access to our Bucket by attaching our newly created Policy.
+
+1. Open [IAM](https://console.aws.amazon.com/iam/home) and navigate to "Users" on the left
+2. Click "Add Users"
+3. Specify a name and make sure to choose "Access key - Programmatic access"
+
+![create user](../../images/docs/apps/s3-export/create-user.png)
+
+4. Click "Next"
+5. At the top, select "Attach existing policies directly"
+6. Search for the policy you just created and click the checkbox on the far left to attach it to this user
+
+![attaching the policy to our newly created user](../../images/docs/apps/s3-export/attach-policy.png)
+
+6. Click "Next" until you reach the "Create user" button. Click that as well.
+7. **Make sure to copy your "Access key" and "Secret access key". The latter will not be shown again.**
+
+![showing our newly created api key and secret key](../../images/docs/apps/s3-export/access-keys.png)
+
+### Step 3: Add data to the bucket
+
+> For this section, we'll be using **Airbyte**. However, we accept any data in CSV or Parquet format, so if you already have data in S3 you can skip this section.
+
+1. Go to [Airbyte](https://airbyte.com) and sign up for an account if you haven't already.
+2. Go to connections and click "New connection"
+3. Select a source. For this example we'll be grabbing data from **Stripe**, but you can use any of Airbyte's sources.
+4. Click "Set up a new destination"
+5. Select "S3" as the destination
+6. Fill in the "S3 Bucket Name", "S3 Bucket Region" with the name and region you created earlier.
+7. For "S3 Bucket Path", use `airbyte`.
+8. For the "Output Format", pick Parquet. You can use the default settings
+9. Under "Optional fields", you'll want to add the access key and secret from step 1.
+10. In the next step, pick the streams you want to fill. Given you'll manually need to create a table for each stream, we suggest being selective.
+11. Wait for the sync to finish
+
+![creating the s3 destination](../../images/features/data-warehouse/airbyte-destination.png)
+
+### Step 4: Create the table
+
+1. In PostHog, go to the "Data Warehouse" tab and click "New table"
+2. Fill in the table name.
+3. For the URL pattern, copy the URL from s3, up to the bucket name. Replace the file name with `*`, as Airbyte will split larges streams out into multiple files.
+  - For example: `https://your-bucket.s3.amazonaws.com/airbyte/invoices/*`
+4. For file format, select Parquet. Fill in the access key and secret key.
+
+> You'll want to repeat this step for each "stream" or folder that Airbyte created in your S3 bucket.
+
+
+![creating table in posthog](../../images/features/data-warehouse/create-table.png)
+
+### Step 5: Query the table.
+
+Amazing! You can now query your new table in any HogQL query.
```

Comment:
```suggestion
Amazing! You can now query your new table in any [HogQL query](/docs/product-analytics/hogql).
```
1. An example would be nice. Maybe a code block with an SQL insight that lists transactions from Stripe.
2. Can users now find this data in the database tab of data management? If so, we should mention and link to there.